### PR TITLE
Force django to load the wsgi on startup

### PR DESCRIPTION
**Pull request by NyanKiyoshi**
This avoids latencies when the workers are serving their first request coming from Django lazy-loading the application.

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [x] The changes are tested.
1. [ ] GraphQL schema and type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by django-queries** [comment]
Here is the report for d292963cccbd0bea5126e22c54413313fc58abd5 (NyanKiyoshi/saleor @ uwsgi/warmup)
Base comparison is e4856e4efa2a346d6dd393fef997a28b77d6f81e.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark category
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  category view                              	         23	         23	              0

# api.benchmark checkout mutations
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  add billing address to checkout            	         55	         55	             28
  add shipping to checkout                   	         32	         32	              5
  checkout email update                      	         33	         33	             16
  checkout payment charge                    	         21	         21	              4
  checkout shipping address update           	         29	         29	              2
  checkout voucher code                      	         62	         62	             34
  complete checkout                          	         66	         66	             13
  create checkout                            	        148	        148	             83
  update checkout lines                      	         95	         95	             50

# api.benchmark collection
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  collection view                            	         21	         21	              0

# api.benchmark homepage
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  featured products list                     	         20	         20	              0
  retrieve main menu                         	          8	          8	              0
  retrieve product list                      	          6	          6	              0
  retrieve secondary menu                    	          8	          8	              0
  retrieve shop                              	          2	          2	              0
  user checkout details                      	         53	         53	             28

# api.benchmark order
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  user order details                         	         17	         17	              4

# api.benchmark product
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product details                            	         17	         17	              3
  retrieve product attributes                	          9	          9	              0

# api.benchmark variant
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product variant bulk create                	         48	         48	              3
  retrieve variant list                      	         23	         23	              9

# api.benchmark variant stocks
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product variants stocks create             	         19	         19	              4
  product variants stocks delete             	         16	         16	              4
  product variants stocks update             	         23	         23	              4

# api product sorting attributes
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  sort product not having attribute data     	         20	         20	              0
```

</p>
</details>

File: saleor/wsgi/__init__.py [hunk 1]
```diff
@@ -27,3 +28,16 @@
 # from helloworld.wsgi import HelloWorldApplication
 # application = HelloWorldApplication(application)
 application = health_check(application, "/health/")
+
+# Warm-up the django application instead of letting it lazy-load
+application(
+    {
+        "REQUEST_METHOD": "GET",
+        "SERVER_NAME": "127.0.0.1",
```
Comment (patrys) [inline saleor/wsgi/__init__.py]: This could fail before fully warming up if the name does not match ALLOWED_HOSTS.

File: saleor/wsgi/__init__.py [hunk 2]
```diff
@@ -27,3 +36,16 @@
...
+
+# Warm-up the django application instead of letting it lazy-load
+application(
+    {
+        "REQUEST_METHOD": "GET",
+        "SERVER_NAME": SimpleLazyObject(get_allowed_host_lazy),
+        "REMOTE_ADDR": "127.0.0.1",
+        "SERVER_PORT": 80,
+        "PATH_INFO": "/graphql/",
+        "wsgi.input": sys.stdin,
```
Comment (patrys) [inline saleor/wsgi/__init__.py]: I don't think that's a good idea. If we absolutely need to pass input, let's pass a BytesIO buffer.

**Comment by maarcingebala** [comment]
@patrys Could you re-review this one?

**Pull request merged**